Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
                     </script>
                 </section>
 
-                <section data-markdown>
-                    <script type="text/template">
-                        # Tool development and integration into Galaxy
-                    </script>
+                <!-- INTRODUCTION -->
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## A tool in Galaxy
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### A tool in Galaxy
+
+                            <img src="../images/galaxy_instance_detailed_screenshot.png" alt="Galaxy instance"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Galaxy tool / wrapper
+
+                            <img src="../images/graphlan_screenshot.png" alt="Tool Form" height=400/>
+
+                            ```bash
+                            graphlan.py --format "png" --size 7 input_tree.txt png_image.png
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            - Description of the user interface
+                            - Link between the tool and Galaxy instance
+                                - How to invoke the tool
+                                - What options to pass
+                                - What files the tool will produce as output
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_layers.png" height=400/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" width=500/>
+
+                            > Command-line utilities to assist in building and publishing Galaxy tools.
+
+                            - [Documentation](https://planemo.readthedocs.io/en/latest/)
+                            - [Tutorial](https://planemo.readthedocs.io/en/latest/writing_standalone.html)
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                              ### Questions
```

Comment:
They were broken!
I only search a few ... minutes and decide to remove them. I was lazy.
Sorry for that!
I will undo that and consult you